src/types/UiaTextRangeBase.hpp

```diff
@@ -72,6 +73,8 @@ typedef unsigned int Endpoint;
 
 constexpr IdType InvalidId = 0;
 
+static constexpr std::wstring_view defaultWordDelimiter = UnicodeSpaceStringView;

```

Comment:
i don't see the need for a separate constant that's just another constant with a different name.

Reply:
Won'tFix-ing this. Moved it inside UiaTextRangeBase. That way, if we ever want to change this parameter on the fly, we can just change this var. I just like this architecture better.